src/com/aca/homework/week5/light_out/Game.java

```diff
@@ -0,0 +1,155 @@
+package com.aca.homework.week5.light_out;
+
+import java.util.Scanner;
+
+public class Game {
+    private static int moves = 0;
+    private Light[][] board;
+
+    public Game() {
+        initialize();
+    }
+
+    private void initialize() {
+        boolean stateOff = false;
+        boolean stateOn = true;
+        board = new Light[5][5];
+        for (int i = 0; i < 5; i++) {
+            for (int j = 0; j < 5; j++) {
+                if (isStateOffLightsCoordinates(i, j)) {
+                    board[i][j] = new Light(stateOff);
+                } else {
+                    board[i][j] = new Light(stateOn);
+                }
+            }
+        }
+    }
+
+    private boolean isStateOffLightsCoordinates(int i, int j) {
+        if (i - j == 2 || j - i == 2) {
+            return true;
+        }
+        if (i == 1 && j == 1) {
+            return true;
+        }
+        if (i == 3 && j == 3){
+            return true;
+        }
+        return false;
+    }
+
+
+    public void startGame() {
+        System.out.println("The game started!!");
+        print();
+        Scanner scanner = new Scanner(System.in);
+        do {
+            System.out.println("Please enter a hit coordinates in '(x,y)' format:");
+            String newCoordinates = scanner.next();
+            while (!isRightFormat(newCoordinates)) {
+                System.out.println("Invalid coordinates!!");
+                newCoordinates = scanner.next();
+            }
+            int i = getCoordinate(newCoordinates) % 10;
```

Comment:
instead of returning an integer that you have to parse, you could return an object of Coordinate class